mapa.html

```diff
@@ -0,0 +1,76 @@
+<html>
+  <head>
+    <link rel="stylesheet"
+          href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css"
+          integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A=="
+          crossorigin=""/>
+     <script
+          src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"
+          integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA=="
+          crossorigin=""></script>
+  </head>
+  <body>
+    <style>
+      #mapid { height: 90vh; }
+    </style>
+     <div id="mapid"></div>
+
+    <script>
+      var categories = [
+        { name: "Bibliotecas", icon: "local_library",     subclass: "Q7075" },
+        { name: "Cinemas",     icon: "movie",             subclass: "Q41253" },
+        { name: "Galerias",    icon: "filter_frames",     subclass: "Q1007870" },
+        { name: "Museus",      icon: "museum",            subclass: "Q33506" },
+        { name: "Venues",      icon: "workspaces_filled", subclass: "Q18674739" },
+        { name: "Teatros",     icon: "theater_comedy",    subclass: "Q24354" },
+      ];
+
+      var mymap = L.map('mapid').setView([41.14961, -8.61099], 13);
+
+      L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+            subdomains: ['a', 'b', 'c'],
+            attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors, <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>',
+      }).addTo(mymap);
+
+      async function loadCategoryLayer(category) {
+        const url = `https://query.wikidata.org/bigdata/namespace/wdq/sparql?format=json&query=${encodeURIComponent(`
+          SELECT ?itemLabel ?geo WHERE {
+            ?item wdt:P31/wdt:P279* wd:${category.subclass};
+            wdt:P625 ?geo .
+            ?item wdt:P17 wd:Q45 .
+            SERVICE wikibase:label { bd:serviceParam wikibase:language "pt,en" } .
+          }
```

Comment:
Sugiro usar o formato de query melhorado (primariamente para legibilidade) introduzido em #29 e #31.

```suggestion
          SELECT ?itemLabel ?geo WHERE {
            ?item  wdt:P31/wdt:P279*  wd:Q7075.  # Item is an instance or subclass of the target category
            ?item  wdt:P625           ?geo.      # Store item's geographic coordinates in the ?geo variable
            ?item  wdt:P17            wd:Q45.    # Item's country is Portugal
            
  	        SERVICE wikibase:label { bd:serviceParam wikibase:language "pt,en" }.
          }
```

Reply:
Só vi a tua suggested change depois de já ter actualizado, podes ver se ficou OK?

Reply:
Looks good! 👍